ecosystem/sep-0008.md

```diff
@@ -3,51 +3,61 @@
+Stellar is an ideal platform for issuing securities. Regulation sometimes requires asset issuers to monitor and approve each transaction involving issued assets and to enforce constraints. This is possible on the Stellar network today, however there exists no standard interface between wallets and issuers to negotiate approval.
 
 ## Overview
 
-Implementing a Regulated Asset consists of these parts:
-- **Stellar.toml**: Issuers will advertise the existence of an Approval Service and the approval criteria via their stellar.toml file.
-- **Approval Server**: HTTP protocol for transaction signing.
-- **Account Setup**: Wallets will have to work with accounts that are controlled or at least partially controlled (via multisig) by asset issuers, rather than the wallet end user.<br/>
-**Note**: this step may not be required once the proposed [protocol change](https://github.com/stellar/stellar-protocol/issues/146) allowing for protocol-level per-transaction approval is implemented.
+Implementing a regulated asset requires these parts:
+- Issuer account with appropriate [authorization flags] set.
+- [SEP-1 stellar.toml] used for discovering [Approval Server].
+- [Approval Server] that validates client transactions according to the service's approval criteria. Validated transactions are signed by the asset's issuing account.
 
 ## Regulated Assets Approval Flow
 
-1. User creates and signs a transaction.
+1. Wallet creates and signs a transaction.
 2. Wallet resolves asset information and detects that it's a regulated asset.
+	1. Wallet can detect whether an asset is a regulated asset by checking the [authorization flags] of the asset issuer's account.
 3. Wallet sends the transaction to the approval server.
-4. Approval server determines whether the transaction is compliant based on the current state of the ledger, known pending transactions, and its compliance ruleset.
+	1. Wallet can find the approval server via [SEP-1 stellar.toml] set up by the issuer.
+4. Approval server determines whether the transaction should be approved.
 5. Wallet handles approval response:
-    1. *Success?* Transaction has been approved and signed by issuer. Submit to Horizon.
-    2. *Revised?* Transaction has been revised to be made compliant, and signed by the issuer. Wallet will show the changes to the user and ask them to sign the new transaction. Submit to Horizon once signed.
-    3. *Pending?* The issuer will need to asynchronously approve this transaction. Wallet should send the same transaction to the approval service after a specified timeout (Return to 3).
+    1. *Success?* Transaction has been approved and signed by the issuer.
+    2. *Revised?* Transaction has been revised to be made compliant, and signed by the issuer. Wallet will show the changes to the user and ask them to sign the new transaction. It is important to understand that an approval service could respond with a different transaction.
```

Comment:
Lets add bold for emphasis
```suggestion
    2. *Revised?* Transaction has been revised to be made compliant, and signed by the issuer. Wallet will show the changes to the user and ask them to sign the new transaction. **It is important to understand that an approval service could respond with a different transaction.**
```

Reply:
https://github.com/stellar/stellar-protocol/pull/813/commits/ad9d8ad85441646c268aca250df77fcb27c2d2c0